src/api/app/models/workflow/step.rb

```diff
@@ -43,7 +46,7 @@ def create_or_update_subscriptions(package, workflow_filters)
                                                           enabled: true,
                                                           token: @token,
                                                           package: package)
-      subscription.update!(payload: scm_webhook.payload.merge({ workflow_filters: workflow_filters, short_package_name: target_package_name(short_commit_sha: true) }))
```

Comment:
We don't use `short_package_name` anymore in `ReportToScmJob` where this payload is used, so this why we remove this key from the payload.